### PR TITLE
Bump `unicode-loose-match`

### LINKED ISSUE #71: CJK Unified Ideographs Extension I block is not covered in `15.1.0-scripts.txt`
The CJK Unified Ideographs Extension I block `2EBF0..2EE5D` is sorta last-minute changes included in the Unicode 15.1 (c.f. [L2/23-114R](https://www.unicode.org/L2/L2023/23114r-unc-extension-i.pdf)). It was not included in the draft fetched as of March 2023. 

There might be other changes that are not in my radar, I suggest we fetch the UCD again and update the data.

### PR DESCRIPTION
**Pull request by JLHwung**
Actually bump `unicode-loose-match` and fix the assertion guard when `unicode-loos-match` is out of dated.

Also added a minimal GH test workflow.

Fixes #71

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    test:
+        name: Test on Node.js Latest
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
+            - name: Use Node.js latest
+              uses: actions/setup-node@v3
+              with:
+                node-version: latest
```
Comment (mathiasbynens) [inline .github/workflows/ci.yml:18]: Let’s keep `.nvmrc` as the single source of truth, i.e. update the version number in it and then refer to it here:

```suggestion
      - name: Set up Node.js
        uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
```

(also let’s use two-space indents?)

**Review by mathiasbynens — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            - name: Install dependencies
+              run: npm install
+            - name: Run tests
+              run: npm run test
```
Comment (mathiasbynens) [inline .github/workflows/ci.yml:22]:
```suggestion
              run: npm test
```

**Comment by JLHwung** [comment]
@mathiasbynens 🏓

**Review by mathiasbynens — APPROVED** [review]

**Comment by mathiasbynens** [comment]
Thanks!

**Pull request merged**